ansible_collections/arista/avd/molecule/cv_deploy/cv_deploy.yml

```
@@ -0,0 +1,132 @@
+---
+- name: Converge - Configuration deployment with CVP
+  hosts: SITE1_FABRIC
+  connection: local
+  gather_facts: false
+  vars:
+    cv_server: www.cv-staging.corp.arista.io
+    cv_token: "{{ lookup('env', 'CVAAS_AAWG_CI') }}"
+    cv_verify_certs: false
```

As we are hitting public CVaaS endpoint - we should let SSL to verify server cert to make sure we are talking to a valid CVaaS endpoint
```suggestion
    cv_verify_certs: true
```